apps/web/app/hooks/useCollaborative.ts

```diff
@@ -0,0 +1,98 @@
+import { IUser } from '@app/interfaces';
+import {
+	collaborativeMembersState,
+	collaborativeSelectState,
+} from '@app/stores';
+import { useCallback } from 'react';
+import { useRecoilState } from 'recoil';
+import { useAuthenticateUser } from './features/useAuthenticateUser';
+import { useOrganizationTeams } from './features/useOrganizationTeams';
+import { BOARD_APP_DOMAIN } from '@app/constants';
+import { useRouter } from 'next/router';
+import { generateRandomString } from '@app/helpers';
+
+export function useCollaborative(user?: IUser) {
+	const { activeTeam } = useOrganizationTeams();
+	const { user: authUser } = useAuthenticateUser();
+	const [collaborativeSelect, setCollaborativeSelect] = useRecoilState(
+		collaborativeSelectState
+	);
+	const [collaborativeMembers, setCollaborativeMembers] = useRecoilState(
+		collaborativeMembersState
+	);
+
+	const url = useRouter();
+
+	const randomMeetName = useCallback(() => generateRandomString(15), []);
```

Comment:
I would also say that meet name can include for example names of people who were invited or if no one invited, include name of person who created it. Plus those random characters generated with that lib @badalkhatri0924 suggesting :)

Reply:
@evereq if user invite 10 people it will have 10 names! I think better we have random value only (In my opinion).

Reply:
Where that value will be used? If only in the URL, then it's OK to use only random value, but if it will be used as a Room Name in UI, I would prefer to somehow have better names for such Rooms? @paradoxe35 ?

Reply:
Okay @evereq 
Gonna add these changes